TestProjects/HDRP_Tests/ProjectSettings/EditorBuildSettings.asset

```diff
@@ -287,9 +287,6 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/GraphicTests/Scenes/2x_Lighting/2212_ReflectionProbes_Skies.unity
     guid: c193bc6b9e5a3764a92da565cdf8d74d
-    enabled: 1
```

Comment:
you shouldn't have remove this test :). Note that we also need to keep it in order (there was an issue with XR if we switch order)